src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,78 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_ITEMS_QUANTITY = 10;
+    private K key;
+    private V value;
+    private StorageImpl[] storages;
```

Comment:
```suggestion
    private Pair[] elements;
```
Создай внутренний класс Pair.